deegree-core/deegree-core-base/pom.xml

```diff
@@ -38,10 +38,6 @@
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
     </dependency>
-    <dependency>
```

Comment:
Why removing this specific xerces dependency?

Reply:
xerces is used only in deegree-core-commons.